src/components/Alert.tsx

```diff
@@ -16,27 +16,25 @@ import Notification from './Notification'
 import RedoIcon from './RedoIcon'
 import UndoIcon from './UndoIcon'
 
-/** An alert component that fades in and out. */
-const Alert: FC = () => {
-  const alert = useSelector(state => state.alert)
-  const alertStoreValue = alertStore.useState()
-  const value = strip(alertStoreValue ?? alert?.value ?? '')
-  const iconSize = useSelector(state => 0.78 * state.fontSize)
-  const multicursor = useSelector(state => state.alert?.alertType === AlertType.MulticursorActive)
-  const dispatch = useDispatch()
-
+/** A custom hook that manages a delayed effect with start and clear timer functions. The callback will not be called if delay is 0 or undefined. */
+const useDelayedEffect = (callback: () => void, delay: number | undefined, trigger?: unknown) => {
```

Comment:
Are you sure that `trigger` is actually needed? Would be better with just `callback` and `delay` args.